src/tests/ftest/util/mpio_utils.py

```diff
@@ -70,14 +66,18 @@ def run_romio(self, basepath, hostlist, romio_test_repo):
         """
 
         # environment variables only to be set on client node
-        env_variables = ["export CRT_ATTACH_INFO_PATH={}/install/tmp/" \
-                             .format(basepath),
-                         "export MPI_LIB=''", "export DAOS_SINGLETON_CLI=1"]
+        env = {
+            "CRT_ATTACH_INFO_PATH": "{}/install/tmp/".format(basepath),
+            "MPI_LIB": "\"\"",
+            "DAOS_SINGLETON_CLI": 1,
+            "D_LOG_FILE": os.environ.get("D_LOG_FILE", ""),
+            "OFI_INTERFACE": os.environ.get("OFI_INTERFACE", "eth0")
+        }
 
         # setting attributes
-        cd_cmd = 'cd ' + romio_test_repo
-        test_cmd = './runtests -fname=daos:test1 -subset -daos'
-        run_cmd = cd_cmd + ' && ' + test_cmd
+        run_cmd = exports_cmd(env) + os.path.join(romio_test_repo,
```

Comment:
Suggestion to utilize existing class in `util/command_utils.py`:
```
env = EnvironmentVariables()
env["CRT_ATTACH_INFO_PATH"] = "{}/install/tmp/".format(basepath)
env["MPI_LIB"] = "\"\""
env["DAOS_SINGLETON_CLI"] = 1
env["D_LOG_FILE"] = os.environ.get("D_LOG_FILE", "")
env["OFI_INTERFACE"] = os.environ.get("OFI_INTERFACE", "eth0")

test_cmd = [
    env.get_export_str(),
    os.path.join(romio_test_repo, 'runtests'),
    '-fname=daos:test1',
    '-subset',
    '-daos']
run_cmd = " ".join(test_cmd)
```

Reply:
Fixed.